src/Book/Book.tsx

```diff
@@ -80,6 +97,29 @@ const Book = (props: Props) => {
             .catch(err => console.log(err))
     }
 
+    const notifySuccess = (expirationDate : string) => toast.success('Solicitud de prestamo satisfactoria. Recuerde que debe devolverlo antes del ' + expirationDate, toastifyConfigurations);
+
+    const notifyError = (message : string) => toast.error(message, toastifyConfigurations);
+
+    const handleLoan = (book : Book) => {
+        const promise =  post(`loan/ ${book.id}`, {});
```

Comment:
The is an extra space between the `loan/` and `${book.id}` this adds an unwanted `%` into the URL.
Please delete the extra space.